integration/examples/helm-deployment/skaffold.yaml

```diff
@@ -8,7 +8,7 @@ build:
 deploy:
   helm:
     releases:
-    - name: skaffold-helm
+    - name: skaffold-helm-{{.TEST_NS}}
```

Comment:
Aren't the examples synced to integration from time to time? In that case, the namespace suffix would be removed.

Reply:
AFAIK, the script we have just makes sure for each directory in example, we have a directory in integration test.
I don't think we sync them from time to time. 
Having this in examples might break examples if users don't have it set.